GameEngine/Players/Player.cs

```diff
@@ -21,11 +22,24 @@ public void Discard(CardType card)
             Hand.Remove(card);
         }
 
-        public abstract CardType SelectCardToPlay(GameBoard board);
-
         public bool HandContainsSun()
         {
             return Hand.Contains(CardType.Sun);
         }
+
+        public abstract Play FormulatePlay(GameBoard board);
+
+        protected int PositionOfStragglerOwl(GameBoard board)
```

Comment:
💡 These two don't really feel to me like they belong here. It feels a bit premature to guess what other `Players` will want in terms of helpers.

Reply:
These have been moved to the `Parliament` class.